character.py

```diff
@@ -24,6 +25,7 @@ def talk_to_player(self, player, player_msg):
                 else:
```

Comment:
If you're implementing my suggestions (see below) here you'll want to initialize your Item with however many points you want.